DivisionManagementSystem/settings.py

```diff
@@ -84,6 +84,29 @@
 
 WSGI_APPLICATION = 'DivisionManagementSystem.wsgi.application'
 
+# Logging
```

Comment:
This comment just duplicates the name of the config variable. I guess it follows an established style in this file, but that style is a bit shit.

It might be better to actually explain what this config variable does, and maybe link to relevant documentation in the django docs for anyone who wants to change it in the future.

Reply:
Very good point, me 6 months from now will probably appreciate that a lot. Fixed.